android/clear_environment.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+set -e
+
+# Script to clear environment BOINC for Android
+
+if [ ! -d ../../boinc/android ]] ; then
+    echo You run this script from diffrent folder: $PWD
+    echo Please run it from boinc/android
+    exit 1
+fi
+
+rm -rf ../3rdParty/buildCache/
```

Comment:
What is the purpose to remove everything from this folder? It could contain cache for different architectures.
I'd remove only ${PREFIX} folder but it should be already cleaned up from `android/buildAndroidBOINC-CI.sh` script

Reply:
I'm afraid, this is still unresolved:
- here we have path to cache that could be changed by parameter
- in case if this folder contains cache of other architectures, it will be removed too, and I don't think that it's good

Reply:
Maybe the content of this file could be safely moved to `android/buildAndroidBOINC-CI.sh` ? then it can safely use set paths.

Reply:
This is a local script. It resolve many problem in one click.
I think it not should merge with the clean.
Just leave it intact like it, and if someone has a problem, it will help it to fix it.

Reply:
i can remove it from do clean from android/buildAndroidBOINC-CI.sh